api/v1alpha1/etcd_types.go

```diff
@@ -103,6 +103,24 @@ type CompressionSpec struct {
 	Policy *CompressionPolicy `json:"policy,omitempty"`
 }
 
+// OwnerCheckSpec defines parameters related to checking if the seed is an owner
+// of the shoot. The ownership can change during control plane migration.
```

Comment:
Can you define this in a general way, i.e. no Gardener architecture terms? etcd-druid can be used w/o Gardener and I also think the ownership check can be usable outside of the Gardener context as well, right?